app/presenters/tree_builder_belongs_to_hac.rb

```diff
@@ -0,0 +1,89 @@
+class TreeBuilderBelongsToHac < TreeBuilder
+  has_kids_for Datacenter, [:x_get_tree_datacenter_kids, :type]
+  has_kids_for EmsCluster, [:x_get_tree_cluster_kids]
+  has_kids_for EmsFolder, [:x_get_tree_folder_kids]
+  has_kids_for ExtManagementSystem, [:x_get_kids_provider]
+  has_kids_for ResourcePool, [:x_get_resource_pool_kids]
+
+  def override(node, object, _pid, _options)
```

Comment:
You should not use underscored arguments such as `_options` when actually using the argument. We have this as a convention: if the argument is unused/optional, it should begin with an underscore. Otherwise just remove the underscore and use it normally, thanks.